redaxo/src/addons/structure/plugins/content/lib/article_content_base.php

```diff
@@ -232,6 +271,10 @@ protected function correctValue($value)
         return $value;
     }
 
+    /**
+     * @param string $value
+     * @return string|int|null
```

Comment:
kommen floats als string?

Reply:
ich hab den typ aus `rex_article` übernommen.
ich denke, die tabellen haben keine float-spalten.
(wenn es welche gäbe, würden sie zurzeit als string kommen, aber ab php 8.1 nicht mehr)